app/build.gradle.kts

```diff
@@ -14,6 +14,14 @@ plugins {
   `crowdin-plugin`
 }
 
+repositories {
+  // https://github.com/chrisbanes/tivi/blob/main/build.gradle
+  val composeSnapshot = libs.versions.composeSnapshot.get()
+  if (composeSnapshot.length > 1) {
+    maven("https://androidx.dev/snapshots/builds/$composeSnapshot/artifacts/repository/")
```

Comment:
Needs a `content` block like the repositories defined in `BaseProjectConfig.kt`. Use `includeGroupByRegex("androidx\\.compose\\.*")` as the inclusion rule.